### DISCUSSION TITLE
Some Github TLS certificates are coming across as expired

### DISCUSSION THREAD

**Discussion by cmars** [community/community#50966] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Some Github TLS certificates are coming across as expired

$ openssl s_client -connect raw.githubusercontent.com:443
CONNECTED(00000006)
depth=2 C = US, O = DigiCert Inc, OU = www.digicert.com, CN = DigiCert Global Root CA
verify return:1
depth=1 C = US, O = DigiCert Inc, CN = DigiCert TLS RSA SHA256 2020 CA1
verify return:1
depth=0 C = US, ST = California, L = San Francisco, O = "GitHub, Inc.", CN = *.github.io
verify error:num=10:certificate has expired
notAfter=Mar 21 23:59:59 2023 GMT

**Comment by cmars** [comment]
Other hosts are affected

ERROR: cannot verify objects.githubusercontent.com's certificate, issued by ‘CN=DigiCert TLS RSA SHA256 2020 CA1,O=DigiCert Inc,C=US’:
  Issued certificate has expired.

(that's in circle CI, so I know it's not just happening in my network neck of the woods)

**Reply by brownz11** [reply]
We're seeing this too, has broken some CD pipelines in Terraform

**Reply by FernandoSena1** [reply]
Não acredito qui ele demorou tanto pra fazer esse lixo

**Comment by jurajseffer** [comment]
Community UI broken because of expired/invalid cert for `avatars.githubusercontent.com` too

**Comment by ghost** [comment]
Both raw. and www. have expired certificates too, hope they get to it soon.

**Comment by maxemann96** [comment]
`support.github.com` is also expired 2 days ago

**Comment by WebFreak001** [comment]
issue also affecting connections from some GitHub actions / CI runners. I had an issue on my own connection, but randomly works sometimes, might be some distribution / mirror thing.

**Comment by AroKol78** [comment]
```
Timestamp: 2023-03-24 21:59:50
Error: avatars.githubusercontent.com:443 uses an invalid security certificate.

The certificate expired on 22 marca 2023 00:59. The current time is 24 marca 2023 21:59.

(Error code: SEC_ERROR_EXPIRED_CERTIFICATE)
```

---
it's already working for me 🚀

**Comment by cmars** [comment]
Also, docs.github.com has the wrong CN:

```
% openssl s_client -connect docs.github.com:443
CONNECTED(00000006)
depth=2 C = US, O = DigiCert Inc, OU = www.digicert.com, CN = DigiCert Global Root CA
verify return:1
depth=1 C = US, O = DigiCert Inc, CN = DigiCert TLS RSA SHA256 2020 CA1
verify return:1
depth=0 C = US, ST = California, L = San Francisco, O = "GitHub, Inc.", CN = *.githubassets.com
```

**Comment by tasansal** [comment]
I am having the same issue.

```shell
docker pull ghcr.io/org/package
```

causes:

```shell
failed to copy: httpReadSeeker: failed open: failed to do request: Get "https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:*": x509: certificate is valid for *.githubassets.com, githubassets.com, not pkg-containers.githubusercontent.com
```

**Comment by danielbraun89** [comment]
And [githubstatus.com](https://www.githubstatus.com/) is all green. Right

**Reply by tjlauer** [reply]
That's because it's fixed now

**Reply by rekkih** [reply]
> That's because it's fixed now

tbf it was green even when everything was having issues, including the status page itself 😅

**Comment by alexanderhoet** [comment]
I get the following error when trying to pull an image from ghcr.io:

`error pulling image configuration: download failed after attempts=6: x509: certificate is valid for *.githubassets.com, githubassets.com, not pkg-containers.githubusercontent.com`

Seems related to this topic.

**Reply by alexanderhoet** [reply]
It's working again!!

**Comment by developedsoftware** [comment]
curl: (60) SSL: no alternative certificate subject name matches target host name 'pkg-containers.githubusercontent.com'

This must be affecting everything lol!

**Comment by retr0h** [comment]
Looks like the cert has been corrected @ my geo (los angeles)

**Comment by pbilstein** [comment]
Also breaks spring bootBuildImage.. and probably many other processes depending on this service

**Comment by tasansal** [comment]
Working now at Houston, TX

**Comment by SvenLauterbach** [comment]
![why-is-it-when-something-happens-its-always-you-three](https://user-images.githubusercontent.com/1760777/227641793-389fb47d-a05c-4ddf-9b0b-589c755b5403.png)

**Reply by pbilstein** [reply]
BGP ftw

**Comment by retr0h** [comment]
/me hey Siri set a reminder to vendor my k3s images

🤣

**Comment by SvenLauterbach** [comment]
Works in Munich too, again, thanks

**Comment by ghost** [comment]
Works in Amsterdam, good work everyone :clap:

**Comment by ch4rms** [comment]
Confirmed working here too.
https://www.githubstatus.com/incidents/x7njwb481j9b Maybe a little bigger impact than just `Pages` ;)
